### PR TITLE
Add `parse-lsblk` plugin

### PR DESCRIPTION
**Pull request by benjamreis**
The plugin returns the equivalent of running `lsblk --json` on a host

Signed-off-by: BenjiReis <benjamin.reis@vates.fr>

### CODE REVIEW

File: SOURCES/etc/xapi.d/plugins/parse-lsblk.py
Comment (Wescoeur) [inline SOURCES/etc/xapi.d/plugins/parse-lsblk.py:23]: You can use `parent` directly instead of `parent.keys()`.

File: SOURCES/etc/xapi.d/plugins/parse-lsblk.py
Comment (stormi) [inline SOURCES/etc/xapi.d/plugins/parse-lsblk.py:11]: Why shell=True?

Comment (benjamreis) [reply SOURCES/etc/xapi.d/plugins/parse-lsblk.py:11]: `lsblk` doesn't work without it.

**Review by stormi — APPROVED** [review]
Approved, provided we squash the commits before the merge of course.

**Review by Wescoeur — APPROVED** [review]

**Pull request merged**
